lib/fugue/regula.rego

```diff
@@ -229,11 +232,65 @@ merge_reports(reports) = ret {
   }
 }
 
+# Waiver globbing.
+waiver_pattern_matches(pattern, value) {
+  pattern == "*"
+} {
+  pattern == value
+}
+
+# Should a rule-resource result be waived?
+waiver_matches_rule_result(rule_result) {
+  waiver := data.fugue.regula.config.waivers[_]
+  waiver_resource_id := object.get(waiver, "resource_id", "*")
```

Comment:
Is there a condition somewhere to filter out empty waivers? Not sure how often that'd come up in practice, but I think the behavior we'd want is that an empty waiver would have no effect rather than waiving everything.